blog/02-05-2023-integrating-ipfs-with-react/integrating-ipfs-with-react.md

```diff
@@ -0,0 +1,581 @@
+---
+title: 'Decentralized Storage and Seamless User Experience: Integrating IPFS with React for Your ERC721 Smart Contract'
+description: This tutorial would guide you through the basics of IPFS and how you can integrate it in your React project
+authors:
+  - name: Daniel Ogbuti
+    title: Web3 Developer,
+    url: https://github.com/dahnny
+    image_url: https://github.com/dahnny.png
+tags: ["celo", "intermediate", "celosage","ipfs"]
+hide_table_of_contents: true
+slug: /tutorials/integrating-ipfs-with-react
+---
+
+![header](../../src/data-tutorials/showcase/intermediate/integrating-ipfs-with-react.png)
+
+
+## Introduction
+
+Welcome! Welcome!! Welcome!!! to another tutorial, This tutorial is part of a bigger NFT series and the aim of this is to help one become an expert in building NFT marketplaces as we know, NFTs are the future.
+In this tutorial, we would be going through the basics of NFT and how to implement it in your react project.
+We would be explaining parts of a bigger project and may leave some parts out that might not be relevant to this tutorial.
+
+Following the project, we would need to store our images on IPFS so we could use that for the image URL as in the contract we would use for this NFT pathway.
+
+### MyNFT.sol
+
+```js
```

Comment:
please adjust the formatting for the programming language to  solidity instead of js